uber/site_sections/preregistration.py

```diff
@@ -191,12 +191,31 @@ def banned(self, id):
         attendee, group = self._get_unsaved(id)
         return {'attendee': attendee}
 
+    def freereg_payment(self, session):
+        charge = Charge(listify(self.unpaid_preregs.values()))
+        if charge.total_cost == 0:
+            for attendee in charge.attendees:
+                attendee.paid = c.HAS_PAID
```

Comment:
Remove this line and the next -- it makes no sense in this context. Attendees who did not pay should not be marked as having paid.